lfs/gitscanner_refs.go

```diff
@@ -105,6 +106,45 @@ func scanMultiLeftRightToChan(scanner *GitScanner, pointerCb GitScannerFoundPoin
 	return scanRefsToChan(scanner, pointerCb, []string{refLeft}, bases, gitEnv, osEnv, opt)
 }
 
+// scanRefsByTree scans through all commits reachable by refs contained in
+// "include" and not reachable by any refs included in "excluded" and invokes
```

Comment:
Technically the variable name is `"exclude"` not `"excluded"`, but this typo also affects the existing comment [above](https://github.com/git-lfs/git-lfs/blob/38ae601d5db0da9f94ee3e822d9bd09d4c39da7b/lfs/gitscanner_refs.go#L35) `scanRefsToChan()`, so it's not new.

(As a sidebar to be above, I happened to see that `scanRefsToChan()`'s comment also [says](https://github.com/git-lfs/git-lfs/blob/38ae601d5db0da9f94ee3e822d9bd09d4c39da7b/lfs/gitscanner_refs.go#L35-L36) that it "returns a channel of `WrappedPointer` objects", but I suspect that hasn't been true since commit 3085dc32c7e52f1b2522a19a9259d1612b8fa4a5 in PR #1743 in 2016, when the function's sole return value became just `error`.)

Reply:
I've updated the comment.